pkg/plugins/plugin.go

```diff
@@ -14,3 +20,42 @@ type VendorPlugin interface {
 	// Apply config change
 	Apply() error
 }
+
+type HostHelpersInterface interface {
```

Comment:
should you call this interface VendorPluginHelpersInterface ? (and matching struct)

or rename the New methods ?

usually we have `New{Type}()` naming convention for such functions